Cargo.toml

```diff
@@ -36,16 +37,19 @@ exclude = [
 bench = false
 
 [features]
-default = ["prost-derive"]
+default = ["prost-derive", "std"]
 no-recursion-limit = []
+std = []
+   # When std is disabled, we attempt to provide no_std support in prost
 
 [dependencies]
-bytes = "0.5"
-prost-derive = { version = "0.6.1", path = "prost-derive", optional = true }
+bytes = { version = "0.5", default-features = false }
+prost-derive = { version = "0.6.1", path = "prost-derive", default-features = false, optional = true }
 
 [dev-dependencies]
 criterion = "0.3"
 env_logger = { version = "0.7", default-features = false }
+failure = { version = "0.1", default-features = false }
```

Comment:
This seems fishy, is it a necessary addition?

Reply:
I'd recommend `displaydoc` as a `no_std`-friendly custom derive that does something similar to `failure_derive`

Reply:
My comment is questioning the addition of the dependency, I don't see why it's necessary.

Reply:
you're probably right, sorry, my bad. I forgot that we moved away from failure in prost library

Reply:
this likely showed up during the merge-resolution